src/Stepper/Stepper.less

```diff
@@ -0,0 +1,25 @@
+@import "../less/fonts.less";
+@import (reference) "../less/index";
+
+.Stepper {
+  .flexbox();
+}
+
+.Stepper--sidebar {
+  .flex(0, 0, 18rem);
+  max-width: 18rem;
+  .padding--l;
+}
+
+.Stepper--stepsDisplay {
+  list-style-type: none;
+  .padding--none;
+
+  li {
+    &:not(:last-child)::after {
+      display: block;
```

Comment:
I recently discovered that there is a `display--block` mixin in clever-components
https://github.com/Clever/components/blob/bb7ea958db87b896b6ec1a225dbb0cc2c4caa347/src/less/display.less#L3

It doesn't save any code, so I don't see an advantage to changing this.